app/services/calculate_status.rb

```diff
@@ -1,20 +1,28 @@
 class CalculateStatus
   UPDATE_TIME = 1
 
+  attr_reader :silent
+  def initialize(silent: Rails.env.test?)
```

Comment:
With additional service specs, this was getting a bit noisy.

Reply:
oh nice, didn't know about this =D